autocomplete/definitions/namedTypes/tes3quest/id.lua

```diff
@@ -0,0 +1,6 @@
+return {
```

Comment:
`TES3::Quest` inherits `TES3::BaseObject`. So a duplicate definition here is not needed.